packages/polymath-issuer/src/pages/account/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  PageCentered,
+  Button,
+  Toaster,
+  notify,
+  Grid,
+} from '@polymathnetwork/ui';
+import { connect } from 'react-redux';
+import { fetchTickerList } from '../../actions/account';
+import TickerCard from './components/TickerCard';
+
+class HomePage extends Component {
+  async componentDidMount() {
+    await this.props.fetchTickerList();
+    if (this.props.tickers.length === 0) {
+      this.props.history.push('/ticker');
+    }
+  }
+
+  render() {
+    const { tickers } = this.props;
+    return (
+      <PageCentered>
+        <Grid>
+          <Grid.Row>
+            <p>
+              <Link to="/ticker">
+                <Button id="create-token-btn" icon="arrow--right">
+                  Continue to create your security token
```

Comment:
@FahdW per our most recent changes in `develop`, the word "Create" should be replaced with "Configure"